### PR TITLE
[13.0][MIG] hr_timesheet_sheet

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Supersedes https://github.com/OCA/timesheet/pull/337.

In the tests, I applied the server-side formview utility.

And also I squashed .pot file and number version update automatically generated commits.

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@MiquelRForgeFlow shouldn't it first be a isort/black/prettier and then migration according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch ?

**Comment by MiquelRForgeFlow** [comment]
Yes, and that's the order implemented. It's just that github show commits ordered by commit creation date instead of correct commit order.

You can check by loading this in local and doing "git log -5", which it will show:

```
...
[FIX] hr_timesheet_sheet: drop fuzzy+python-format translations
[IMP] : black, isort
[MIG] hr_timesheet_sheet: Migration to 13.0
[IMP] hr_timesheet_sheet: black, isort, prettier
[MIG] hr_timesheet_sheet: Migration to 13.0 (complete)
```

**Comment by alexey-pelykh** [comment]
Hmm, github shows different history for some reason

**Comment by MiquelRForgeFlow** [comment]
I can edit the date of the commits to make it look how it should, but I thing it's not a good thing, and also I don't care. We should live with it, and hope someday github change their order of showing the commits.

**Comment by alexey-pelykh** [comment]
Fine with me as it is as long in `git log` it's correct :)

File: hr_timesheet_sheet/security/hr_timesheet_sheet_security.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            <field name="name">Timesheet sheet multi-company</field>
+            <field
+                name="model_id"
+                search="[('model','=','hr_timesheet.sheet')]"
+                model="ir.model"
+            />
+            <field name="global" eval="True" />
+            <field
+                name="domain_force"
+            >['|',('company_id','=',False),('company_id','in',[company_ids])]</field>
```
Comment (astirpe) [inline hr_timesheet_sheet/security/hr_timesheet_sheet_security.xml]:
```suggestion
            >['|',('company_id','=',False),('company_id','in',company_ids)]</field>
```

**Review by astirpe — CHANGES_REQUESTED** [review]

**Review by astirpe — APPROVED** [review]

**Review by alexey-pelykh — APPROVED** [review]
Feels many commits due to multiple migration attempts (by different authors) are not needed, but I just support the effort :)

**Comment by MiquelRForgeFlow** [comment]
I don't understand why travis ci becomes hidden.

**Comment by MiquelRForgeFlow** [comment]
It's green :green_apple:

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional test on local

**Pull request merged**